src/renderer/components/Settings.tsx

```diff
@@ -97,6 +98,45 @@ export default function SettingsUi() {
                 </Switch>
             ))}
 
+            {supportsWindowsTransparency && (
+                <>
+                    <Switch
+                        value={Settings.transparent ?? false}
```

Comment:
wouldn't it make more sense to completely remove this switch and instead merge it into the below select? So that the default state of the select is No transparency and then you can choose one of the options

Reply:
done, i've also added back the full restart disclaimer and the theme disclaimer. as for where people would find them, i don't know either - i'm only aware of the [theme](https://github.com/uwu/Acrylic/blob/v2/app/theme.css) that Acrylic uses as well as any "transparent" themes on BetterDiscord that might more-or-less work if edited properly ([like this one](https://[https://github.com/MeowIce/TranslucencePlus]))